DAS/das/utils/MockSensor.py

```diff
@@ -57,4 +66,8 @@ def gen_single_value(self, average_value):
         sensor_val += average_value * random.uniform(-self.percent_range,
                                                      self.percent_range)
         sensor_val = round(sensor_val, self.decimals)
+        if self.increment and self.single_val:
+            new_value = average_value + abs(average_value - sensor_val)
+            self.average_value = (new_value, )
+            return new_value
```

Comment:
Should this be `self.average_value = new_value` instead of `self.average_value = (new_value, )`?

Reply:
The type of `self.average_value` should be a tuple, `new_value` is a float. The `(new_value, )` syntax is to create a tuple with one element (which is what happens when there is a single value).